docs/configuration/OVERVIEW.md

```diff
@@ -1,10 +1,10 @@
 # Configuration mechanism
 
-The aim of this document is to help developers to implement configuration inside components, in order to be compliant
-with __CMS architecture__. This will give the possibility to __Business Analyst__ to configure the
+The aim of this document is to help developers to implement `Configuration` inside components, in order to be compliant 
```

Comment:
comment from @pginoux-1A:
I think the first thing is not about the CMS, it's about making the application configurable dynamically. The CMS is a way to ease the creation of this configuration override, but that's not mandatory. It would be interesting to develop "When do I want to build configurability in my application and why" and then go on the How which is already part of this file

https://github.com/AmadeusITGroup/otter/pull/1583/files#r1557788837

Reply:
I tried something:
"The aim of this document is to help developers to implement configurability in your application. 

You may need configurability if you are developping components that may have different behaviors or different display depending on who or how there are used. 
Let's take the example of a library with calendar components used by many projects all around the world. 
Depending on their culture, some of the users may want a Gregorian Calendar, others may need the Lunar one. 
You will probably want to make your component configurable at least at build time to meet everyone's need. 

Now, let's imagine the same use case not for a library but for an application. You need to implement some sort of runtime configurability to address everyone's need. That is what Otter's `Configuration` pattern is for.

In this page, you will learn how to implement this pattern and how to extract metadata that will describe how your project can be configured. The metadata json can be used to build your own override application, or as an input for a CMS plugin for example.
@pginoux-1A , @sdo-1A what do you think?